apps/web/src/services/asset-inventory/store/collector-page-store.ts

```diff
@@ -30,14 +29,14 @@ export const useCollectorPageStore = defineStore('collector-page', {
         },
     },
     actions: {
-        async getCollectorList(queryData) {
+        async getCollectorList(queryData?: Query) {
             this.loading = true;
             try {
                 const res = await SpaceConnector.client.inventory.collector.list({
                     query: queryData,
                 });
-                this.totalCount = res.total_count;
                 this.collectors = res.results;
+                this.listCount = res.total_count;
```

Comment:
Question: Why did you change the variable from `totalCount` to `listCount`? I think `totalCount` is easier to understand.